src/content/guides/package-exports.md

```diff
@@ -32,18 +33,20 @@ An example:
     ".": "./main.js",
     "./sub/path": "./secondary.js",
     "./prefix/": "./directory/",
-    "./prefix/deep/": "./other-directory/"
+    "./prefix/deep/": "./other-directory/",
+    "./other-prefix/*": "./yet-another/*/*.js"
   }
 }
 ```
 
-| Module request                | Result                                |
-| ----------------------------- | ------------------------------------- |
-| `package`                     | `.../package/main.js`                 |
-| `package/sub/path`            | `.../package/secondary.js`            |
-| `package/prefix/some/file.js` | `.../package/directory/some/file.js`  |
-| `package/prefix/deep/file.js` | `.../package/other-directory/file.js` |
-| `package/main.js`             | Error                                 |
+| Module request                      | Result                                           |
+| ----------------------------------- | ------------------------------------------------ |
+| `package`                           | `.../package/main.js`                            |
+| `package/sub/path`                  | `.../package/secondary.js`                       |
+| `package/prefix/some/file.js`       | `.../package/directory/some/file.js`             |
+| `package/prefix/deep/file.js`       | `.../package/other-directory/file.js`            |
+| `package/other-prefix/deep/file.js` | `.../package/yet-another/deep/file/deep/file.js` |
```

Comment:
This is honestly a great catch; I did not notice this behavior reading the Node docs but it is right there:

> All instances of `*` on the right hand side will then be replaced with this value

@matthewp do we test for this case in esinstall (and if not, we should)?